src/tools/png2gas/CMakeLists.txt

```diff
@@ -96,16 +96,10 @@ endif()
 ################################################################################
 
 # find libSplash installation
-# prefer static libraries over shared ones (but do not force them)
-set(Splash_USE_STATIC_LIBS ON)
-find_package(Splash 1.6.0 REQUIRED COMPONENTS PARALLEL)
+find_package(Splash 1.7.0 REQUIRED CONFIG COMPONENTS PARALLEL)
 
-if(Splash_FOUND)
-    include_directories(SYSTEM ${Splash_INCLUDE_DIRS})
-    list(APPEND Splash_DEFINITIONS "-DENABLE_HDF5=1")
-    add_definitions(${Splash_DEFINITIONS})
-    set(LIBS ${LIBS} ${Splash_LIBRARIES})
-endif(Splash_FOUND)
+add_definitions(-DENABLE_HDF5=1)
```

Comment:
QUESTION:
Why could you remove the if clause?

Reply:
because it's required in `find_package`